DisCatSharp.ApplicationCommands/ApplicationCommandsExtension.cs

```diff
@@ -66,12 +67,12 @@ public sealed class ApplicationCommandsExtension : BaseExtension
         /// <summary>
         /// List of global commands on discords backend.
         /// </summary>
-        private static List<DiscordApplicationCommand> _globalDiscordCommands { get; set; } = new List<DiscordApplicationCommand>();
+        public static List<DiscordApplicationCommand> _globalDiscordCommands { get; set; } = new List<DiscordApplicationCommand>();
```

Comment:
These should stay private. It is just used for internal checks.

Reply:
They have an invalid state after the startup.
The announced methods are safer.

Reply:
Have changed back to private and switched to using the methods on the client.